src/components/ProvinceDropdown/province.data.js

```diff
@@ -0,0 +1,392 @@
+export default [
+  {
+    code: "TH-10",
+    title_th: "กรุงเทพมหานคร",
```

Comment:
Personally, I tend to use camelCase according to JS style guides. So, the title_th key could have been called thaiTitle or something like that.